doc/source/contributing/contribution_guidelines.rst

```diff
@@ -95,6 +54,53 @@ Licensing
 
+Ways to Contribute
+------------------
+
+- Submit bug reports and feature requests, see `Requesting a feature`_ or `Reporting a bug`_
+- Write code for everyone to use, or share your use cases and examples (as Jupyter Notebooks, scripts, etc, see `Adding Examples`_ or `Add ancillary dataset`_)
+- Submit bug reports and feature requests, see `Reporting a bug`_ or `Requesting a Feature`_
+- Write code for everyone to use
+- `Add ancillary dataset`_
+- Fix typos and improve documentation and testing see  `Improving Documentation and Testing`_
+
+The best way to report a problem, request a feature, find out if others are working on a similar problem or application, or let us know you'd like to contribute some code is to find the *Issues* tab and check if your problem/suggestion has already been reported. If so, please provide any additional information in the ongoing discussion. Otherwise, feel free to create a new issue and submit your problem or suggestions.
+
+
+Requesting a Feature
+--------------------
+Find the *Issues* tab at the top of GitHub repository and click *New Issue* button. Please give your suggestion a clear title and let us know if this is something you'd like to work on and contribute.
+
+
+Reporting a bug
+---------------
+Find the *Issues* tab at the top of GitHub repository and click *New Issue* button. Give your issue a clear title and describe the steps required to recreate it in as much detail as possible. If you can, include a small example that reproduces the error. More information and minimal examples will help us resolve issues faster.
+
+
+Questions and Help
+------------------
+
+Please do not create issues to ask for help. A faster way to reach the community is through our Science/ICESat-2 subcategory on the `Pangeo discourse <https://discourse.pangeo.io/c/science/icesat-2/16>`_ page. We are excited to have you join an existing conversation or start a new post! Please note that a GitHub login is required to post on the discourse page.
+
+Other Resources
+^^^^^^^^^^^^^^^
+- Check out our :ref:`ICESat-2 Open-Source Resources Guide<resource_ref_label>` for a host of tools and code for getting and working with ICESat-2 data
+- The ICESat-2 Hackweek Tutorial repositories are great resources for learning basic Python and development skills, learning about ICESat-2 data, setting up a computational environment, and finding and analyzing ICESat-2 datasets.
+    - 2019 repo: <https://github.com/ICESAT-2HackWeek/ICESat2_hackweek_tutorials>`_
+    - 2020 repo: <https://github.com/ICESAT-2HackWeek/2020_ICESat-2_Hackweek_Tutorials>`_
+    - 2022 repo: <https://github.com/ICESAT-2HackWeek/website2022>`_
```

Comment:
```suggestion
    - 2019 repo: `<https://github.com/ICESAT-2HackWeek/ICESat2_hackweek_tutorials>`_
    - 2020 repo: `<https://github.com/ICESAT-2HackWeek/2020_ICESat-2_Hackweek_Tutorials>`_
    - 2022 repo: `<https://github.com/ICESAT-2HackWeek/website2022>`_
```
I think this will get them to render properly on RTD